readme.md

```diff
@@ -63,6 +63,8 @@ To create a Jenkins job:
 5. Add a **build step**.
 6. Click **Save**.
 
+**Note:** Currently, when creating a Pipeline or Multibranch Pipeline job, the use of a Jenkinsfile is required. Other pipeline scripting methods are not supported.
```

Comment:
```suggestion
**Note:** A Jenkinsfile is required when creating a Pipeline or Multibranch Pipeline job. Other pipeline scripting methods are not yet supported. 
```

Looks good @mhenschke-atl . Just thought that: 

- the first sentence could be simplified by dropping 'the use of'. 
- using 'yet' instead of 'currently' would help to imply that this is coming soon.